plugins/node.d/icinga

```diff
@@ -0,0 +1,82 @@
+#!/bin/sh
+
+##%# family=auto
+
+#%# capabilities=autoconf
+
+ICINGACLI=${ICINGACLI:-$(which icingacli)}
+JQ=${JQ:-$(which jq)}
+
+
+if [ "$1" = "autoconf" ] ; then
+    if [ "$ICINGACLI" != ""  -a "$JQ" != "" ] ; then
```

Comment:
Here I have two suggestions:
* it is quite common to use `[ -z "$foo" ]` for testing an empty string
* `-a` (or `-o`) are [not portable and should be avoided](http://mywiki.wooledge.org/BashPitfalls#A.5B_.22.24foo.22_.3D_bar_.26.26_.22.24bar.22_.3D_foo_.5D)

Thus maybe you want to switch to the following style for a more precise error message?
```shell
if [ ! -x "$ICINGACLI" ]; then
    echo "no (could not find 'icingacli')"
elif [ ! -x "$JQ" ]; then
    echo "no (could not find 'jq')"
else
    echo "yes"
fi
```

Reply:
I've changed this. Reason was that I had some problems with the which output.